src/core/astish.js

```diff
@@ -1,4 +1,4 @@
-let newRule = /(?:([A-Z0-9-%@]+) *:? *([^{;]+?);|([^;}{]*?) *{)|(})/gi;
+let newRule = /(?:([\u0080-\uFFFFa-zA-Z0-9-_%@]+) *:? *([^{;]+?);|([^;}{]*?) *{)|(})/g;
```

Comment:
btw i turned off case-insensitivity here: https://stackoverflow.com/questions/32010/is-regex-case-insensitivity-slower

Reply:
Uh oh! 😬  this adds `+15B` 😲

Reply:
the negative set might be even longer tbh, because you'd have to exclude alphanumerics and the `-_%@` chars and the same unicode range, can't really think of a way to make it shorter. i'd rather have a lib that works correctly according to specs, because 15bytes isn't going to make a difference in the real world, not even if we were still using dialup modems.

Reply:
this is now +8 bytes, due to writing out `\u0080-\uFFFF`. it can be omitted to even shave some bytes but it wouldn't be correct.